.github/workflows/build-and-benchmark-x86.yml

```diff
@@ -16,12 +16,15 @@ jobs:
               with:
                 toolchain: 1.87.0 # specific version for consistent perf
                 target: x86_64-unknown-linux-gnu
+                components: clippy
```

Comment:
It seems like this is not getting installed? Idk what's going on here.

Reply:
It was picking up the `rust-toolchain.toml` instead of the installed toolchain that's forced via the workflow.